wolfssl/wolfcrypt/asn_public.h

```diff
@@ -244,7 +244,7 @@ typedef struct WOLFSSL_ASN1_TIME {
 
 enum {
     IV_SZ   = 32,                   /* max iv sz */
-    NAME_SZ = 80,                   /* max one line */
+    NAME_SZ = 160,                  /* max one line */
```

Comment:
This gets used in many places. Embedded systems may not want this increased. Please gate 80 vs 160 on a build macro.

Reply:
Done.